### DISCUSSION TITLE
When LOTUS_MINER_PATH is NFS, the retrieval speed is much much faster.

### DISCUSSION THREAD

**Discussion by kernelogic** [filecoin-project/lotus#6865] [General Q&A]
### Lotus component

lotus miner/market - retrieval deal

### Lotus Version

Daemon:  1.11.0-rc2+mainnet+git.0519cd371+api1.3.0
Local: lotus version 1.11.0-rc2+mainnet+git.0519cd371

### Describe the Bug

This ticket is a tracking of this discussion discovered by XinAn Xu and confirmed by me. https://filecoinproject.slack.com/archives/C022ZR4JA1M/p1626684614283800

As we all know lotus retrieval is painfully slow, merely 1MB/s on ideal internet connection and nobody knows why yet. 

However XinAn Xu discovered if the provider mount the home folder as NFS, the retrieval speed can be drastically faster, like 50MB/s.

So for example my provider's home folder is /home/lotus/.lotusminer , instead of specifying LOTUS_MINER_PATH=/home/lotus/.lotusminer, I could do the following:

1. export this folder as a NFS mount point in /etc/exports by adding a new line, for example:  
`/home/lotus/.lotusminer 127.0.0.1/32(rw,sync,no_subtree_check,insecure)`

2. Restart NFS server `systemctl restart nfs-kernel-server` 

3. Mount it locally. For example `mount 127.0.0.1:/home/lotus/.lotusminer /mnt/nfsminer`

4. Change env variable `LOTUS_MINER_PATH=/mnt/nfsminer` and restart lotus-miner. 

5. If want to make the mount permanent. add a new line to /etc/fstab:
`127.0.0.1:/home/lotus/.lotusminer /mnt/nfsminer nfs defaults 0 0`

Voila! Now retrieval is at full speed!

This is not an ideal solution of course and we hope lotus dev team can investigate more why NFS can make such a big difference than the native EXT4 file system.

### Logging Information

N/A

### Repo Steps

```bash
N/A
```

**Comment by jennijuju** [comment]
Thanks for testing and sharing! We will take a look!

**Comment by kernelogic** [comment]
Right now XinanXu, WillTechMusing, Nelson, DeeGee and I have been using this to gain retrieval speed successfully.

**Reply by Shekelme** [reply]
But will any other local filesystem (XFS for example) make the similar difference in retrieval speed?
